cmd/queue/main.go

```diff
@@ -448,9 +478,8 @@ func createVarLogLink(servingNamespace, servingPodName, userContainerName, varLo
 	}
 }
 
-func pushRequestLogHandler(currentHandler http.Handler) http.Handler {
-	templ := os.Getenv("SERVING_REQUEST_LOG_TEMPLATE")
-	if templ == "" {
+func pushRequestLogHandler(currentHandler http.Handler, env envConfig) http.Handler {
```

Comment:
make this pass the loggingconfig only vs. passing the whole config.

Reply:
Modified as per the comment